server/auth/auth.go

```diff
@@ -0,0 +1,740 @@
+// Copyright 2020 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissionKeys and
+// limitations under the License.
+
+package auth
+
+import (
+	"encoding/json"
+	"github.com/pingcap/log"
+	"github.com/tikv/pd/pkg/errs"
+	"go.uber.org/zap"
+	"path"
+	"strings"
+	"sync"
+
+	"github.com/pingcap/errors"
+	"github.com/tikv/pd/server/kv"
+	"go.etcd.io/etcd/clientv3"
+)
+
+var (
+	// ErrUserNotFound is error info for user not found.
+	ErrUserNotFound = func(name string) error { return errors.Errorf("user not found: %s", name) }
```

Comment:
I think the error definition should be placed in [here](https://github.com/tikv/pd/blob/master/pkg/errs/errno.go).